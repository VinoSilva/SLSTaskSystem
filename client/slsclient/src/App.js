import React from 'react';

import thunk from 'redux-thunk';

import { createStore,applyMiddleware } from 'redux';
import { Provider } from 'react-redux';

import rootReducer from './reducers/rootReducer'; 

import Task from './Components/Task';

const store = createStore(rootReducer,applyMiddleware(thunk));

//Remove history,test-utils, and remove unnecessary npm

function App() {
  return (
    <Provider store={store}>
      <div className="App">
        <Task />
      </div>
    </Provider>
  );
}

export default App;
