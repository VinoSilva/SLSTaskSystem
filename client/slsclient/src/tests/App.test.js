import React from 'react';
import ReactDOM from 'react-dom';
import App from '../App';

import { shallow } from 'enzyme';

import renderer from 'react-test-renderer';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });


describe('App component', () => {
  
  it('renders hello world', () => {
    const wrapper = shallow(<App />);
    const text = wrapper.find('h1').text();
    expect(text).toEqual('Hello World');

  });

  it('App matches the snapshot',()=>{
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});