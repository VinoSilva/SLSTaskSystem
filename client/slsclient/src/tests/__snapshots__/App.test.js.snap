// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component App matches the snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Hello World
  </h1>
</div>
`;
